addons/html_editor/static/src/main/media/image_transform_button.js

```diff
@@ -7,7 +7,7 @@ export class ImageTransformButton extends Component {
     static template = "html_editor.ImageTransformButton";
     static props = {
         icon: String,
-        getSelectedImage: Function,
```

Comment:
Here I think we need to keep the old name. If we rename it then the old prop won't be called anymore if someone had called it with some other function.